### ISSUE TITLE
ReadMe.MD

### ISSUE THREAD

**Issue by firsturdiev** [zixing131/myjump#3]
Could you please provide more information about the project?

**Comment by zixing131** [comment]
As you can see, all the source code is here.

**Comment by firsturdiev** [comment]
Aren't you going to continue the project? Or it's already done?

**Comment by zixing131** [comment]
It's not over yet, there's still a long way to go, and this project is slowly continuing.

**Comment by zixing131** [comment]
What needs to be done now is to improve compatibility, as well as network support and 3D support. You can pull down the code and directly modify the files in the Java folder before pushing them up. Thank you.

**Comment by firsturdiev** [comment]
Thanks, I'll check them out.

**Comment by firsturdiev** [comment]
Is your project a fork of Mozilla Pluorbet? If yes, what's the main
difference between the two?

**Comment by zixing131** [comment]
You are right,this projcet is  a fork of Mozilla Pluorbet,but this project is direct modify the  compiled project, It add sound support and add some exception ignore to support more jars.
